app/javascript/chat/chat.jsx

```diff
@@ -253,7 +253,7 @@ export default class Chat extends Component {
           activeChannel ||
           this.filterForActiveChannel(channels, activeChannelId),
       });
-      this.setupChannel(activeChannelId);
+      this.setupChannel(activeChannelId, activeChannel);
```

Comment:
@narender2031, are there tests to test this change or does it require some new tests?

Reply:
Just following up on this @narender2031.

Reply:
@nickytonline  There are no existing test cases for mention. we need to add new test cases for the mentions.

Reply:
Just following up on this.

Reply:
> @nickytonline  @citizen428 we require some test cases for this change. Currently, we don't have any test cases for this.  

I see the bad merge has been fixed @sarthology. Thanks. Just following up on the missing tests.